examples/message-transfers.rs

```diff
@@ -0,0 +1,75 @@
+        
+        Curabitur sodales elit eget leo efficitur cursus. Proin eget neque dictum, luctus ipsum sed, porta ex. Morbi eget libero sem. Etiam vitae dignissim nunc. In elementum eu augue eget malesuada. Suspendisse at ornare erat, et convallis elit. Suspendisse rhoncus metus odio, sed ornare quam consectetur ut. Suspendisse a nunc vel leo iaculis varius. Aliquam erat volutpat. Nullam semper vulputate odio, et sagittis nunc ultrices non.
+        
+        Curabitur sit amet sem laoreet, egestas magna in, eleifend orci. Maecenas tincidunt nunc a eros sollicitudin, quis gravida tellus interdum. Curabitur id ipsum diam. Proin at massa urna. Sed neque mauris, efficitur vitae congue nec, posuere a magna. Vestibulum nunc tortor, euismod non faucibus a, aliquet eget sapien. Vivamus eros dui, viverra at mauris a, cursus pulvinar turpis. Vivamus non tempor leo. Donec condimentum nisl a turpis aliquam aliquet. Phasellus sollicitudin ultricies orci, ac ultrices leo. Praesent ultricies, quam accumsan ornare sodales, massa lacus tempus nunc, sed convallis metus dui et nibh.
+        
+        Aenean ultrices porttitor enim. Vestibulum vel ante at neque laoreet sollicitudin ut nec purus. Donec tortor lectus, egestas non gravida nec, consequat eu orci. Sed id vehicula eros. Aliquam leo odio, finibus nec placerat sit amet, dignissim in est. Suspendisse in neque auctor, bibendum nunc non, luctus dolor. Nulla convallis felis ac libero egestas, at egestas nunc consequat. Suspendisse tellus massa, consequat non felis facilisis, maximus convallis tortor. Fusce porta nunc est, et facilisis dolor imperdiet vitae. Vivamus ullamcorper erat et dignissim mattis. Nulla maximus, nisi vel lobortis gravida, arcu neque blandit ante, at sodales risus enim a metus. Nulla quis finibus mauris. ")),
+    });
+
+    // Create a client instance
+    let res = iota::Client::new("https://nodes.comnet.thetangle.org")
+        // Call send_transfers api
+        .send_transfers()
+        // Below is just a dummy seed which just serves as an example.
+        // If you want to replace your own. It probably should be a seed with balance on comnet/devnet.
+        .seed(
+            &IotaSeed::<Kerl>::from_buf(
+                TryteBuf::try_from_str("RVORZ9SIIP9RCYMREUIXXVPQIPHVCNPQ9HZWYKFWYWZRE9JQKG9REPKIASHUUECPSQO9JT9XNMVKWYGVA")
+                    .unwrap()
+                    .as_trits()
+                    .encode::<T1B1Buf>(),
+            ).unwrap(),
+        )
+        // Input the transfers
+        .transfers(transfers)
+        // We are sending to comnet, so mwm should be 10. It's 14 by default if you don't call this.
+        .min_weight_magnitude(10)
+        // Sending to the node and receive the response
+        .send()
```

Comment:
how can I set the tag in the transaction?

Reply:
It's not supported atm. And I think it's better left as is for now since we have been talking tag is not reliable for a while.

Reply:
tag is not reliable on what? afaik, tag is an optional field and it's user-defined.